src/standardMethods.ts

```diff
@@ -225,38 +229,52 @@ export function generateRetrieveSingleEndpoint<IdType, ResponseType>(
 export function generateDeleteEndpoint<IdType>(
   axiosInstance: AxiosInstance,
   resourcePath: string,
-  metadataMap: MetadataMap
+  metadataMap: MetadataMap,
+  chunkFunction?: (ids: IdType[]) => IdType[][]
 ) {
   return async function remove(ids: IdType[]): Promise<{}> {
-    const response = await rawRequest<{}>(
-      axiosInstance,
-      {
-        url: `${resourcePath}/delete`,
-        method: 'post',
-        data: { items: ids },
-      },
+    const chunks = doChunking<IdType>(ids, chunkFunction);
+    const responses = await promiseAllWithData(
+      chunks,
+      input =>
+        rawRequest<ItemsResponse<ResponseType>>(axiosInstance, {
+          method: 'post',
+          url: `${resourcePath}/delete`,
+          data: { items: input },
+        }),
       true
     );
-    return metadataMap.addAndReturn({}, response);
+    return metadataMap.addAndReturn({}, responses[0]);
```

Comment:
I would rather merge with `Object.assign()` all of them to see the actual result.
Although it should not fail

Reply:
What do you mean? `/delete` only returns an empty object...

Reply:
well, that's because we know it.. but, showing just one response doesn't feel good
what if one them wan not empty?

Reply:
The return value here is `{}` and not `responses[0]`. `responses[0]` is used to get the headers/status code from the response (`{}`). But yes, we don't expose all headers from all the requests